seatunnel-connectors-v2/connector-common/src/main/java/org/apache/seatunnel/connectors/seatunnel/common/multitablesink/MultiTableSinkAggregatedCommitter.java

```diff
@@ -58,16 +57,15 @@ private void initResourceManager() {
             }
             resourceManager =
                     ((SupportMultiTableSinkAggregatedCommitter<?>) aggCommitter)
-                            .initMultiTableResourceManager(aggCommitters.size(), 1)
-                            .orElse(null);
+                            .initMultiTableResourceManager(aggCommitters.size(), 1);
             break;
         }
         if (resourceManager != null) {
             for (String tableIdentifier : aggCommitters.keySet()) {
                 SinkAggregatedCommitter<?, ?> aggCommitter = aggCommitters.get(tableIdentifier);
                 aggCommitter.init();
                 ((SupportMultiTableSinkAggregatedCommitter<?>) aggCommitter)
-                        .setMultiTableResourceManager(Optional.of(resourceManager), 1);
+                        .setMultiTableResourceManager(resourceManager, 0);
```

Comment:
why update 1 - > 0 ?

Reply:
It is index of committer, so it should be started with 0.